UI/window-basic-stats.cpp

```diff
@@ -389,6 +394,38 @@ void OBSBasicStats::Update()
 
 	outputLabels[0].Update(strOutput, false);
 	outputLabels[1].Update(recOutput, true);
+
+	totalSeconds += (TIMER_INTERVAL / 1000);
+
+	long double kbps = outputLabels[1].kbps;
+
+	if (kbps == 0.0l) {
+		recordTimeLeft->setText("N/A");
+		return;
+	}
+
+	bitrates.push_back(kbps);
+
+	if (totalSeconds % 30 != 0)
```

Comment:
Probably makes sense to do a check here if seconds is > 30 and then reset it. This enables the TIMER_INTERVAL to change without breaking this and also we don't forever increment a variable.

Reply:
I'm now using a QTimer instead.